benchmarks/dashboard-stack/lib/dashboard-stack.ts

```diff
@@ -1,22 +1,67 @@
 import * as cdk from '@aws-cdk/core';
 import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
 import * as iam from '@aws-cdk/aws-iam';
+import * as _lambda from '@aws-cdk/aws-lambda';
+import * as codebuild from '@aws-cdk/aws-codebuild';
+import * as events from '@aws-cdk/aws-events';
+import * as targets from '@aws-cdk/aws-events-targets';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as s3deploy from '@aws-cdk/aws-s3-deployment'
 import * as path from 'path';
 import * as fs from 'fs';
 
 export class DashboardStack extends cdk.Stack {
     constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
         super(scope, id, props);
 
-        const benchmark_config_json = fs.readFileSync(path.join(__dirname, 'benchmark-config.json'), 'utf8');
+        const benchmark_config_json_path = path.join(
+            __dirname,
+            "..",
+            "..",
+            "benchmark-config.json"
+        );
+        const benchmark_config_json = fs.readFileSync(benchmark_config_json_path, 'utf8');
         const benchmark_config = JSON.parse(benchmark_config_json);
 
+        // Lambda to handle trigger codebuild to deploy benchmark and clean up benchmark after tests
+        const lambda_role = iam.Role.fromRoleArn(this, 'LambdaRole', 'arn:aws:iam::123124136734:role/lambda-role-CFN');
+        const lambda = new _lambda.Function(this, 'BenchmarkManager',
+            {
+                runtime: _lambda.Runtime.PYTHON_3_8,
+                code: _lambda.Code.fromAsset("lambda"),
+                handler: "benchmarkManager.benchmarkManager",
+                role: lambda_role,
+                functionName: "BenchmarkManager",
+                timeout: cdk.Duration.minutes(15)
+            });
+
+
+        // Creat the Cloudwatch Event to schedule the lambda to run daily
+        const lambda_target = new targets.LambdaFunction(lambda, {
+            event: events.RuleTargetInput.fromObject({ 'action': 'test' })
+        });
+
+        new events.Rule(this, 'ScheduleRule', {
+            schedule: events.Schedule.rate(cdk.Duration.days(1)),
+            targets: [lambda_target],
+            ruleName: 'BenchMarksTrigger',
+            description: 'Trigger BenchMarks test.'
+        });
+
         let region = 'unknown';
 
         if (props != undefined && props.env != undefined && props.env.region != undefined) {
             region = props.env.region;
         }
 
+        const vpc = new ec2.Vpc(this, 'VPC', {
+            enableDnsSupport: true,
+            enableDnsHostnames: true
+        })
+
+        cdk.Tags.of(vpc).add('source-ag:environment-type', 'prodVPC');
```

Comment:
Hmmmm why "prodVPC" as the tag?

Reply:
Some copy/pasted from an example😅, will change to something more meaningful